File: src/lessons/recursion/spiral/SpiralUseEntity.c
//RemoteTurtle

void spiral(int steps, int angle, int length, int increment)	{
	if (steps <= 0) {
		return;
	} else {
		forward(length);
		left(angle);
		spiral(steps-1, angle, length+increment, increment);
	}
}

/* BEGIN TEMPLATE */
void run() {
	spiral(100,91,1,2);
}
/* END TEMPLATE */

File: resources/exercises/recursion/logo/spiral/SpiralUseEntity.py
def spiral(steps, angle, length, increment):
  if (steps <= 0):
    pass
  else:
    forward(length)
    left(angle)
    spiral(steps-1, angle, length+increment, increment)
  
# BEGIN TEMPLATE
spiral(100,91,1,2)
# END TEMPLATE
